### ISSUE TITLE
importing @clr/angular in an angular cli library Project causes a Unexpected value 'undefined' error

### ISSUE THREAD

**Issue by aavelyn** [vmware/clarity#2536]
```
[x ] bug
[ ] feature request
[ ] enhancement
```

### Expected behavior
running ng build --prod draft should produce a library output.

package.json of library project
```json
{
  "name": "draft",
  "version": "0.0.1",
  "peerDependencies": {
    "@angular/common": "^6.0.0-rc.0 || ^6.0.0",
    "@angular/core": "^6.0.0-rc.0 || ^6.0.0",
    "@clr/angular": "^0.12.0",
    "@clr/icons": "^0.12.0",
    "@clr/ui": "^0.12.0",
    "moment": "^2.22.2",
    "rxjs": "^6.0.0",
    "rxjs-compat": "^6.2.2",
    "@webcomponents/custom-elements": "^1.2.0",
    "lodash": "^4.17.10",
    "object-fit-images": "^3.2.3"
  }
}
```

Inside root/projects/draft/tsconfig.lib.json

```
{
  "extends": "../../tsconfig.json",
  "compilerOptions": {
    "outDir": "../../out-tsc/lib",
    "target": "es2015",
    "module": "es2015",
    "moduleResolution": "node",
    "declaration": true,
    "sourceMap": true,
    "inlineSources": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "importHelpers": true,
    "types": [],
    "lib": [
      "dom",
      "es2015",
      "es2017.object"
    ],
    "paths": {
      "@angular/*": [
        "../node_modules/@angular/*"
      ],
      "@clr/*":[
        "../node_modules/@clr/*"
      ]
    }
  },
  "angularCompilerOptions": {
    "annotateForClosureCompiler": true,
    "skipTemplateCodegen": true,
    "strictMetadataEmit": true,
    "fullTemplateTypeCheck": true,
    "strictInjectionParameters": true,
    "flatModuleId": "AUTOGENERATED",
    "flatModuleOutFile": "AUTOGENERATED"
  },
  "exclude": [
    "src/test.ts",
    "**/*.spec.ts"
  ]
}

```

Inside root/projects/draft/src/draft.module
```js
import { ClarityModule } from '@clr/angular';
```

### Actual behavior

BUILD ERROR
: Unexpected value 'undefined' exported by the module 'ClrFormsNextModule in draft/node_modules/@clr/angular/clr-angular.d.ts'

Error: : Unexpected value 'undefined' exported by the module 'ClrFormsNextModule in draft/node_modules/@clr/angular/clr-angular.d.ts'

    at Object.<anonymous> (draft/node_modules/ng-packagr/lib/ngc/compile-source-files.js:53:68)
    at Generator.next (<anonymous>)
    at draft/node_modules/ng-packagr/lib/ngc/compile-source-files.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (draft/node_modules/ng-packagr/lib/ngc/compile-source-files.js:3:12)
    at Object.compileSourceFiles (draft/node_modules/ng-packagr/lib/ngc/compile-source-files.js:19:12)
    at Object.<anonymous> (draft/node_modules/ng-packagr/lib/ng-v5/entry-point/ts/compile-ngc.transform.js:26:32)
    at Generator.next (<anonymous>)
    at draft/node_modules/ng-packagr/lib/ng-v5/entry-point/ts/compile-ngc.transform.js:7:71
    at new Promise (<anonymous>)

### Reproduction of behavior

* Include a link to the reproduction scenario you created by forking one of the Clarity Plunker Templates:
  
  

### Environment details

* **Angular version:** 6.0.3

* **Clarity version:** 0.12.0

* **OS and version:** macOS 10.13.6

* **Browser:**  Chrome 68

**Comment by gnomeontherun** [comment]
Is this a duplicate of https://github.com/vmware/clarity/issues/2437?

**Comment by aavelyn** [comment]
@gnomeontherun While not the same case, yes. I've solved it by updating to the latest 0.12.8 version.
I was on the 0.12.3 pre patch version.
